src/Tooltip/tooltip.css

```diff
@@ -48,52 +33,66 @@
 
     .tooltipContainer
     {
-        visibility: hidden;
-        position:   absolute;
-        width: var( --tooltipMaxWidth );
-        z-index: 99999;
+        visibility:     hidden;
+        position:       absolute;
+        width:          var( --tooltipMaxWidth );
+        line-height:    0;
     }
 
     .tooltip
     {
         visibility:     visible;
         position:       relative;
-        text-align:     left;
-        box-sizing:     border-box;
         display:        inline-block;
-        min-width:      calc( var( --caretPosOffset ) * 2 );
-        min-height:     calc( var( --caretPosOffset ) * 2 );
+
+        box-sizing:     border-box;
+        min-width:      var( --tooltipMinSize );
+        min-height:     var( --tooltipMinSize);
+        max-width:      var( --tooltipMaxSize );
+
+        border:         var( --tooltipBorderWidth ) solid;
         border-radius:  var( --tooltipBorderRadius );
-        border:         1px solid var( --PC-GREY--D75 );
-        box-shadow:     var( --boxShadow );
-        padding:        var( --spacing-1 ) var( --spacing-2 );
+        box-shadow:     var( --tooltipBoxShadow );
 
-        background:     var( --PC-GREY--D45 );
+        padding:        var( --spacing-2 );
 
-        &::before,
-        &::after
+        text-align:     left;
+
+        &::before /* hover area */
         {
             content:    "";
             position:   absolute;
+            display:    block;
         }
-        &::after
+
+        &::after /* caret */
         {
-          position: absolute;
-          width: var( --caretSize );
-          height: var( --caretSize );
-          background: inherit;
-          border: inherit;
-          border-bottom-color: transparent;
-          border-right-color: transparent;
+            content:                "";
+            display:                block;
+            position:               absolute;
 
-        }
+            box-sizing:             inherit;
+            width:                  var( --caretSize );
+            height:                 var( --caretSize );
+
+            border:                 inherit;
+            border-top-color:       transparent;
+            border-left-color:      transparent;
+
+            background:             inherit;
+         }
```

Comment:
Nice! Almost ready to merge 💣 
One major 🔬 issue here: linter is complaining about 1 extra space here... 😄

Reply:
Turns out I didn’t have `stylelint` enabled 🙄 Thanks for catching!